kratos/input_output/vtk_output.cpp

```diff
@@ -610,6 +643,28 @@ void VtkOutput::WriteVectorSolutionStepVariable(
 /***********************************************************************************/
 /***********************************************************************************/
 
+template<typename TContainerType>
+void VtkOutput::WriteFlagContainerVariable(
+    const TContainerType& rContainer,
+    const Flags Flag,
+    const std::string& rFlagName,
+    std::ofstream& rFileStream) const
+{
+//     mrModelPart.GetCommunicator().SynchronizeFlag(Flag); // TODO: Add syncronization of flags to communicator
```

Comment:
this is implemented now

@jcotela should it be `SynchronizeOrNodalFlags` or `SynchronizeAndNodalFlags`? I am not sure

Reply:
I think that one normally wants SynchronizeOr (flag will be set to true if it is true in any of the local copies of the node), but this could ultimately be a case-specific choice.
Maybe I should add a Synchronize that sets (or unsets) it to whatever is the flag value in the owner process?